Clustergram default values (#324)
plotly/dash-bio: CHANGELOG.md

  figure.
* Fixed issue with Speck not rendering unless it is attached to a callback.
* Prevent Speck from trying to calculate a system with no atom.

> * Prevent Clustergram from not rendering when optional row and/or column

Reviewer: ```suggestion
* Let Clustergram render by passing default values for row and/or column
```

> * Let Clustergram render even when default row and column labels are not 

Reviewer: ```suggestion
* Let Clustergram render even when row and column labels are not
```

>   labels have not been specified.

Reviewer: ```suggestion
  labels.
```
Let's be descriptive and positive rather than double negative (plus verb should be preterit) :)

>   specified by adding default values. 

Reviewer: ```suggestion
  specified, by adding default values.
```

Reviewer: I'm suggesting this additional comma because the following confusion could arise: "labels are not specified by adding default values" vs "let Clustergram render by adding default values" (and we mean the latter).
